pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -73,6 +74,14 @@ type logInfo struct {
 	protocolStr string // protocol of the traffic logged
 }
 
+// logDedup will be used as 1 sec buffer for log deduplication
+type logDedup struct {
+	count       int64       // record count of duplicate log
+	dupLogInfo  string      // initial log used for deleting buffer
```

Comment:
`initial log used for deleting buffer` is a confusing comment. Maybe say something in the lines of: `dupLogInfo` is used as an identifier for unique log entries?